### PR TITLE
Show number of books uploaded from BUB2 to internet archive

### LINKED ISSUE #41: Show number of books uploaded from BUB2 to internet archive
Internet Archive has advanced search API that has an extensive search feature. Search for uploaded by "bub.wikimedia".
Show it in the new /stats route below the queue information. Style accordingly.

### PR DESCRIPTION
**Pull request by BhuwanChandra**
Resolves #41

### CODE REVIEW

File: components/UploadedItems.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  fetchData() {
+    fetch(finalURL)
+      .then((response) => response.json())
+      .then((responseJson) => {
+        const resultyt = responseJson.response.numFound;
+        this.setState({ result: (resultyt-1) });
```
Comment (coderwassananmol) [inline components/UploadedItems.js]: Just curious, why are you decrementing the value?

File: components/UploadedItems.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        console.error(error);
+      });
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  render() {
+    return (
```
Comment (coderwassananmol) [inline components/UploadedItems.js]: Please use a better font and place the text in middle. Increase the font-size as well.
Use this text:
<a href="https://archive.org/details/@bub_wikimedia"><b>{No. of books}</b></a> uploaded to Internet Archive using BUB2!

File: components/UploadedItems.js (hunk 1 above)
Comment (BhuwanChandra) [reply components/UploadedItems.js]: I think, in the uploads, it is also showing the account as 1 uploaded item. so, I need to decrement 1 from the total uploads

File: components/UploadedItems.js (hunk 2 above)
Comment (BhuwanChandra) [reply components/UploadedItems.js]: do I also need to add the link to the archive page of bub.wikimidia?

File: components/UploadedItems.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+var items = 50;
```
Comment (rkpattnaik780) [inline components/UploadedItems.js]: Do we really need this variable?

File: components/UploadedItems.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+var finalURL = `https://archive.org/advancedsearch.php?q=bub.wikimedia+&fl%5B%5D=identifier&fl%5B%5D=publicdate&fl%5B%5D=title&sort%5B%5D=publicdate+desc&sort%5B%5D=&sort%5B%5D=&rows=${items}&page=1&output=json`;
```
Comment (rkpattnaik780) [inline components/UploadedItems.js]: Make a get request to https://archive.org/advancedsearch.php?q=bub.wikimedia+&rows=0&output=json . Current url contains lot of query params that we do not require and the response has the info about uploaded books (which we don't need).

Comment (BhuwanChandra) [reply components/UploadedItems.js]: OK, I'll change that

File: components/UploadedItems.js (hunk 3 above)
Comment (BhuwanChandra) [reply components/UploadedItems.js]: Initially, I thought that I have to show the names of the books uploaded but then I realize it's only the no. the books. I'll fix that

File: components/UploadedItems.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+
+var finalURL = `https://archive.org/advancedsearch.php?q=bub.wikimedia+&rows=0&output=json`;
```
Comment (coderwassananmol) [inline components/UploadedItems.js]: Make this const.

File: components/UploadedItems.js (hunk 1 above)
Comment (coderwassananmol) [reply components/UploadedItems.js]: No. I uploaded a file with that name when I was testing the API. Please remove the decrement.

File: components/UploadedItems.js (hunk 2 above)
Comment (coderwassananmol) [reply components/UploadedItems.js]: Yes. This link: https://archive.org/details/@bub_wikimedia
That's what I added in the above comment.

File: components/UploadedItems.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <div 
+          style={{
```
Comment (coderwassananmol) [inline components/UploadedItems.js]: Don't use style. Use <style jsx> for internal components styling.

File: components/UploadedItems.js (hunk 1 above)
Comment (coderwassananmol) [reply components/UploadedItems.js]: Also, use the `<Link>` component to add a link. Don't use the "a" tag. I have used `<Link>` in the code. You can see it from there. :)

File: components/UploadedItems.js [hunk 7]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
+import Link from "next/link";
 
-var finalURL = `https://archive.org/advancedsearch.php?q=bub.wikimedia+&rows=0&output=json`;
+const finalURL = `https://archive.org/advancedsearch.php?q=bub.wikimedia+&rows=0&output=json`;
```
Comment (rkpattnaik780) [inline components/UploadedItems.js]: Why it has been named finalURL? Let's have only a `url` as variable name.

File: components/UploadedItems.js [hunk 8]
```diff
@@ -16,7 +17,7 @@ class UploadedItems extends Component {
       .then((response) => response.json())
       .then((responseJson) => {
         const resultyt = responseJson.response.numFound;
```
Comment (rkpattnaik780) [inline components/UploadedItems.js]: Variable names should be intuitive/self-suggestive. change it to `booksCount` or something similar. Change the state variable name as well.

**Pull request merged**